tests/test_component.py

```diff
@@ -325,98 +356,6 @@ def get_template(self, context):
             """,
         )
 
-    @djc_test(parametrize=PARAMETRIZE_CONTEXT_BEHAVIOR)
-    def test_prepends_exceptions_with_component_path(self, components_settings):
```

Comment:
The diff in Github is not great, but:
- `test_prepends_exceptions_with_component_path` was moved at the end of `TestComponentRender`
- `test_get_component_by_id` and `test_get_component_by_id_raises_on_missing_component` actually remained here unde `TestComponent`.
- `test_component_render_id` and [`test_input`](https://github.com/django-components/django-components/pull/1180/files#diff-b35cc8d48a3ea2ddde0c3bc9de0b0b768c567f9e8f532dc4ca1b260fe06e4ac8L286) (the latter not highlighted by the diff) were actually moved to own [TestComponentRenderAPI](https://github.com/django-components/django-components/pull/1180/files#diff-b35cc8d48a3ea2ddde0c3bc9de0b0b768c567f9e8f532dc4ca1b260fe06e4ac8R306)